packages/blade/src/components/Table/TableBody.web.tsx

```diff
@@ -0,0 +1,346 @@
+            backgroundColor: getIn(theme.colors, tableRow.stripeWrapper.backgroundColorFocus),
+          },
+          '& tr:nth-child(even):active:not(.disabled-row) .cell-wrapper': {
+            backgroundColor: getIn(theme.colors, tableRow.stripeWrapper.backgroundColorActive),
+          },
+          '& .row-select-single-selected:nth-child(even) .cell-wrapper, .row-select-selected:nth-child(even) .cell-wrapper': {
+            backgroundColor: getIn(theme.colors, tableRow.stripeWrapper.backgroundColorSelected),
+          },
+          '& .row-select-single-selected:nth-child(even):hover:not(.disabled-row) .cell-wrapper, .row-select-selected:nth-child(even):hover:not(.disabled-row) .cell-wrapper': {
+            backgroundColor: getIn(
+              theme.colors,
+              tableRow.stripeWrapper.backgroundColorSelectedHover,
+            ),
+          },
+          '& .row-select-single-selected:nth-child(even):focus:not(.disabled-row) .cell-wrapper, .row-select-selected:nth-child(even):focus:not(.disabled-row) .cell-wrapper': {
+            backgroundColor: getIn(
+              theme.colors,
+              tableRow.stripeWrapper.backgroundColorSelectedFocus,
+            ),
+          },
+          '& .row-select-single-selected:nth-child(even):active:not(.disabled-row) .cell-wrapper, .row-select-selected:nth-child(even):active:not(.disabled-row) .cell-wrapper': {
+            backgroundColor: getIn(
+              theme.colors,
+              tableRow.stripeWrapper.backgroundColorSelectedActive,
+            ),
+          },
+
+          '& tr:nth-child(even):hover:not(.disabled-row) td': {
+            backgroundColor: getIn(theme.colors, tableRow.stripe.backgroundColorHover),
+          },
+          '& tr:nth-child(even):focus:not(.disabled-row) td': {
+            backgroundColor: getIn(theme.colors, tableRow.stripe.backgroundColorFocus),
+          },
+          '& tr:nth-child(even):active:not(.disabled-row) td': {
+            backgroundColor: getIn(theme.colors, tableRow.stripe.backgroundColorActive),
+          },
+
+          '& .row-select-single-selected:nth-child(even) td, .row-select-selected:nth-child(even) td ': {
+            backgroundColor: getIn(theme.colors, tableRow.stripe.backgroundColorSelected),
+          },
+          '& .row-select-single-selected:nth-child(even):hover:not(.disabled-row) td, .row-select-selected:nth-child(even):hover:not(.disabled-row) td ': {
+            backgroundColor: getIn(theme.colors, tableRow.stripe.backgroundColorSelectedHover),
+          },
+          '& .row-select-single-selected:nth-child(even):focus:not(.disabled-row) td, .row-select-selected:nth-child(even):focus:not(.disabled-row) td ': {
+            backgroundColor: getIn(theme.colors, tableRow.stripe.backgroundColorSelectedFocus),
+          },
+          '& .row-select-single-selected:nth-child(even):active:not(.disabled-row) td, .row-select-selected:nth-child(even):active:not(.disabled-row) td ': {
```

Comment:
Where are these class names come from? the react table library?

Reply:
Yup